lib/ui/views/groups/group_details_view.dart

```diff
@@ -14,6 +14,7 @@ import 'package:mobile_app/ui/views/groups/edit_group_view.dart';
 import 'package:mobile_app/ui/views/groups/update_assignment_view.dart';
 import 'package:mobile_app/utils/snackbar_utils.dart';
 import 'package:mobile_app/utils/validators.dart';
+import '../../components/cv_flat_button.dart';
```

Comment:
Please use absolute path

Reply:
Check for requested changes in latest commits.